scripts/background.js

```diff
@@ -1,13 +1,17 @@
-chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
+chrome.tabs.onUpdated.addListener(async (tabId, changeInfo, tab) => {
```

Comment:
I switched this to async when using the browser API namespace, but it's possible that with chrome this is unnecessary in Firefox as well.